sycl/test-e2e/Matrix/SG32/joint_matrix_half_accumulator.cpp

```diff
@@ -1,25 +1,35 @@
-//==----- joint_matrix_bfloat16_packedB.cpp  - DPC++ joint_matrix----------==//
+//==-------SG32/joint_matrix_half_accumulator.cpp  - DPC++ joint_matrix ----==//
 //
 // Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: target-spir
+// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
+// UNSUPPORTED: gpu-intel-dg2
```

Comment:
I'd prefer something like:
```suggestion
// UNSUPPORTED: gpu-intel-dg2
// UNSUPPORTED-INTENDED: WIP - https://github.com/intel-innersource/drivers.gpu.unified/pull/195521
```
Same for the second test. Then you don't have to increase the number and add these tests to the list.

Reply:
I have changed your suggestion so I can merge from github level: GSD-4181 - JIRA for SG32